### ISSUE TITLE
Occasionally acquit-ignore leaks into markdown output

### ISSUE THREAD

**Issue by vkarpov15** [vkarpov15/acquit-ignore#1]
See mongoose-int32

**Comment by hasezoey** [comment]
i dont think this applies anymore, because with https://github.com/vkarpov15/mongoose-int32/pull/14 i have tried updating acquit-ignore to 0.2 and it does not seem to leak anymore (also likely will not leak anymore with https://github.com/vkarpov15/acquit-ignore/pull/6 merged)
